templates/nextflow-launch.yaml

```diff
@@ -0,0 +1,204 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: NextFlow Tower Launch
+Parameters:
+  EcsSecurityGroupId:
+    Type: AWS::EC2::SecurityGroup::Id
+    Description: Security group ID for ECS cluster to grant database access
+  SubnetIds:
+    Type: CommaDelimitedList
+    Description: List of public Subnet Ids for the load balancer
+  MaxvCpus:
+    Type: Number
+    Description: The number of max CPUs for Batch compute environments
+    MinValue: 1
+    MaxValue: 16
+    Default: 1
+Resources:
+  UserManagedPolicy:
```

Comment:
this is setup from the [configure AWS batch manually docs](https://install.tower.nf/22.3/advanced-topics/manual-aws-batch-setup/#create-a-user-policy) however I'm not sure how it's used.  I'm wondering whether this is needed at all?  would the **user** actually be using the [nextflow-launch-iam-policy.yaml](https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/blob/main/templates/nextflow-launch-iam-policy.yaml) instead?